.github/workflows/create_reports.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+ignore="--ignore-not-found"
+logsDir="logs"
+
+createResourceReport () {
+  namespace=$1
+  resource=$2
+  withLogs=$3
+
+  mkdir "$logsDir/$namespace/$resource"
```

Comment:
you will probably need the `-p` flag to create a chain of folders

Reply:
no need for that, here we always create just one additional folder at time

Reply:
ah you're right 👍